js/site.js

```diff
@@ -20,12 +40,17 @@ class Site{
     getScore(){
         this.score = this.scoreFunction();
         return this.score;
-    }
+    };
 
-    getTrackers(){ return this.trackers };
+    _checkGlobalWhitelist(domain){
```

Comment:
When the site object is created this looks up the whitelist status in settings. 
https://github.com/duckduckgo/chrome-zeroclickinfo/pull/81/files#diff-79ef31daa309ba634a1e5919065d33aaR6